.github/workflows/semantic-commit-lint.yml

```diff
@@ -10,19 +10,13 @@ permissions:
 jobs:
   semantic-commit-pr-title-lint:
     runs-on: ubuntu-latest
-    env:
```

Comment:
For now, I'll remove the label, as we are not using it yet and slows down the feedback loop (as we need to pull the repo first). 
We can think about restoring it if we have a workflow that actually uses the label in the future